tilequeue/queue/multisqs.py

```diff
@@ -114,7 +114,11 @@ def read(self, max_to_read=None):
                 except (TypeError, ValueError):
                     timestamp = None
 
-                coord_message = CoordMessage(coord, qm, timestamp)
+                metadata = dict(
+                    queue_name=sqs_queue.name,
+                    timestamp=timestamp,
+                )
+                coord_message = CoordMessage(coord, qm, metadata)
```

Comment:
Yea, I thought it was a better approach to just pass in a more general bag of metadata through on messages instead of explicitly handling each one in the worker. The only other one in use was the timestamp of when the message was added to sqs, so this would supersede that and add the queue name.

Are you on a plane now? :)